osu.Game/Screens/Select/Details/BeatmapDetailsGraph.cs

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2007-2017 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using OpenTK.Graphics;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace osu.Game.Screens.Select.Details
+{
+    public class BeatmapDetailsGraph : FillFlowContainer<BeatmapDetailsBar>

```

Comment:
So I do something like `Rotation = 90` in `BarGraph` and `Bar` updates and changes to it's correct position and size